shared/src/main/scala/com/raquo/domtypes/defs/tags/MathMlTagDefs.scala

```diff
@@ -0,0 +1,334 @@
+package com.raquo.domtypes.defs.attrs
+
+import com.raquo.domtypes.common.{MathMLTagType, TagDef}
+
+
+object MathMLTagDefs {
+
+    val defs: List[TagDef] = List(
+
+      TagDef(
+        tagType = MathMLTagType,
+        scalaName = "math",
+        domName = "math",
+        isVoid = false,
+        scalaJsElementTypeAlias = "org.scalajs.dom.MathMLElement",
```

Comment:
Other tag types spell out this alias without the `org.scalajs.` prefix here, so you should remove it here as well.